### DISCUSSION TITLE
Webcam

### DISCUSSION THREAD

**Discussion by kyuuroneko** [lezzthanthree/Needy-Streamer-Overload#14] [Ideas]
can we get an even bigger option for the webcam, please? 🙏 ame looks too smol on big screen! here is demonstration of how it looks on my monitor and the red lines would be a size suggetion!

![image](https://github.com/lezzthanthree/Needy-Streamer-Overload/assets/152455657/6f94f1c6-6fca-4812-a23d-3feb21f9980a)

**Comment by lezzthanthree** [comment]
You can now adjust Ame Skin's width! Check the latest release for the update!
